### ISSUE TITLE
Windows 10: 0 Call to undefined function mcrypt_get_iv_size()

### ISSUE THREAD

**Issue by nkamp** [vdm-io/Joomla-Component-Builder#509]
Hello,

### Steps to reproduce the issue
When I go to Component builder Dashboard --> Joomla Components and I press "New" or choose the Demo component  I get the error:

**0 Call to undefined function mcrypt_get_iv_size()
Return to controlpanel (button)**

I have download the php_mcrypt.dll and also libmcrypt.dll, put these in the PHP main directory add to the php.ini:
extension=php_mcrypt.dll. This doesn't work so I replace php_mcrypt.dll with libmcrypt.dll but still the same error.

### Expected result
Don't get this error so that I can use it.

### Actual result
-

### System information (as much as possible)
- OS Name & Version:  Windows 10
- MySql Version:  8.0.18
- Apache Version: 2.4
- PHP Version: 7.4
- Joomla Version: 3.9.14
- JCB Version: 2.10.9. Also tried an older version: 2.9.14
- Browser: FF, google chrome

### Additional comments
On this moment, because I have read this as well, I have not add the php_mycript.dll to the registry of windows. It seems to me that it also should work if I should add it to php.ini. I have read this also many times.

What for me hard to understand is, I think I'm not the only one or...

With regards,

Nico

**Comment by MastersOfMedia** [comment]
Hello Nico,

Have you checked if mcrypt is actually loaded by  PHP? For instance by running phpinfo() (you could check the System / System information / PHP information tab in the Joomla! control panel as well), or running the command that shows the active PHP modules from the command line?

This looks more like an issue with your (development) setup than with JCB itself.

Best regards

Erik

**Comment by nkamp** [comment]
Thanks Erik,

But that was the issue and I knew that. 
But than to know which one to install and not download the one which give you more troubles or add something to the registry...

But I think, who is using this component for the first time, should get this issue or...? So I was hoping that someone would say you are doing this wrong.

But after a lot of searching I found it here to solve it:
[start](https://www.php.net/mcrypt)
[Next](https://www.php.net/manual/en/install.pecl.php)
[DLL](https://pecl.php.net/package/mcrypt)
[Download](https://pecl.php.net/package/mcrypt/1.0.3/windows)

Then I made two mistakes in php.ini but with php.exe -i I found them....

At least It's working now! And I can go futher.
Maybe someone else can do his advantage with it.

Nico
